### ISSUE TITLE
SDK examples fail to compile in keystone project

### ISSUE THREAD

**Issue by Dbof** [keystone-enclave/keystone#236]
**EDIT**
I manually compiled the SDK following the README using ``sdk/build`` as build folder and now the compilation from the keystone build folder worked. So maybe this is an issue with the Make files?

**Describe the bug**
Can not compile keystone 'dev' branch following the [official docs](http://docs.keystone-enclave.org/en/latest/Getting-Started/QEMU-Compile-Sources.html). During the first "make" step, compilation errors out on the SDK examples. Basically, any of the examples do not compile (``make examples``)

**Build Failure**
If you ran into build problems, please add these information
(1) Results of `git submodule status`

>  9624a001f9d4314f8bcc9219df717d4c41fb5373 buildroot (2020.11)
 3d77e6a8804abcc0504c904bd6e5cdf3a5cf8162 linux (v5.7)
 0a1597bc6215f00b8c39ff44565df541d74ac782 linux-keystone-driver (v1.0.0)
 fdd76fecdde1ad444ff4deb7f1c4f7e4a1ef97d6 qemu (fdd76fecdd)
 407cdc0ceb0f6d760342ac6cf1b890378f5a81bd riscv-gnu-toolchain
 24a5ed369ac0606aac486382a0855a459d9362ed sdk (v1.0.0)
 843d3509b8a514d43729d779a0cf6bdb1d0c94e0 sm (v1.0)

(2) Your Linux distribution (e.g., Ubuntu 14.04)
Linux Mint 20.1
(3) The branch you're working on (e.g., `master` or `dev`)
dev

**Screenshots or Error Log**
> make -f examples/tests/CMakeFiles/malloc.dir/build.make examples/tests/CMakeFiles/malloc.dir/build
make[3]: Entering directory '/media/user/KEYSTONE/keystone/build2'
[  1%] Building C object examples/tests/CMakeFiles/malloc.dir/malloc/malloc.c.o
cd /media/user/KEYSTONE/keystone/build2/examples/tests && /media/user/KEYSTONE/keystone/riscv64/bin/riscv64-unknown-linux-gnu-gcc  -I/media/user/KEYSTONE/keystone/sdk/build/include  -Wall -Werror   -o CMakeFiles/malloc.dir/malloc/malloc.c.o   -c /media/user/KEYSTONE/keystone/sdk/examples/tests/malloc/malloc.c
/media/user/KEYSTONE/keystone/sdk/examples/tests/malloc/malloc.c:5:10: fatal error: app/eapp_utils.h: No such file or directory
    5 | #include "app/eapp_utils.h"
      |          ^~~~~~~~~~~~~~~~~~
compilation terminated.
make[3]: *** [examples/tests/CMakeFiles/malloc.dir/build.make:66: examples/tests/CMakeFiles/malloc.dir/malloc/malloc.c.o] Error 1
make[3]: Leaving directory '/media/user/KEYSTONE/keystone/build2'
make[2]: *** [CMakeFiles/Makefile2:1575: examples/tests/CMakeFiles/malloc.dir/all] Error 2

Error is the same for other examples including "app/eapp_utils.h".

**Comment by dayeol** [comment]
Did you install the SDK? (`fast-setup.sh` script installs the SDK automatically)
If your `KEYSTONE_SDK_DIR` is the same as SDK's source directory, please do the followings:
```
unset KEYSTONE_SDK_DIR
./fast-setup.sh
```

**Comment by dayeol** [comment]
Any updates? I will close the issue after a couple of days.

**Comment by Dbof** [comment]
Hey there! Sorry, I did not get any email notification for your first response and therefore missed it. Yes, SDK was installed through ``fast-setup.sh`` and the env vars are correctly set by sourcing the generated ``source.sh`` file.

My steps are the same as in the docs:

```
./fast-setup.sh
source ./source.sh
mkdir build
cd build
cmake ..
make
```

Just tried it again with the current dev branch on another machine and it worked. I need to dig more into why it won't build properly on my machine. Closing this for now, thanks for your time!
